### DISCUSSION TITLE
Migrating from Elasticsearch

### DISCUSSION THREAD

**Discussion by mehrdad-khojastefar** [openobserve/openobserve#2852] [Q&A]
Hello, we have an ELK stack and recently we've been interested in replacing our stack with Openobserve but I have some questions that I need to answer before fully migrate to the OpenObserve.
First, how can we restore data in OpenObserve ? Think of the situation like this. We are storing data in s3 in parquet format and now we want to import this data to another OpenObserve instance. How can we do that ? 
Another one is that can we user our existing elastalert as our alert manager? since it is more feature rich than OpenObserve's alerting system. 
Thanks in advance.

**Comment by prabhatsharma** [comment]
Restoration

OpenObserve has 2 things
- Data - stored in s3 in parquet
- Metadata - Stored in sqlite/mysql/postgres/deprecated(etcd, sled, dynamodb)

As long as you are able to provide both data and metadata to the new OpenOserve you will be able to transfer everything.

**Comment by mehrdad-khojastefar** [comment]
You have any ideas about  integrating ElastAlert with openobserve using zplane ?

**Reply by prabhatsharma** [reply]
We have not tested OpenObserve with ElastAlert but it should work. If you face any issue, then let us sknow.
